charts/amoro/values.yaml

```diff
@@ -124,8 +124,30 @@ optimizer:
       jobUri: "local:///opt/flink/usrlib/optimizer-job.jar"
     properties: {
       "flink-conf.taskmanager.memory.managed.size": "32mb",
-      "flink-conf.taskmanager.memory.netwrok.max": "32mb",
-      "flink-conf.taskmanager.memory.netwrok.nin": "32mb"
+      "flink-conf.taskmanager.memory.network.max": "32mb",
+      "flink-conf.taskmanager.memory.network.min": "32mb"
+    }
+  # spark optimizer
+  spark:
+    ## enable the spark optimizer.
+    enabled: true
+    ## container name, default is spark
+    name: ~
+    ## image information of spark optimizer.
+    image:
+      ## the image repository
+      repository: arctic163/optimizer-spark
+      ## the image tag, if not set, the default value is the same with amoro image tag.
+      tag: ~
+      jobUri: "local:///opt/spark/usrlib/optimizer-job.jar"
+    properties: {
+      "master": "k8s://https://<k8s-apiserver-host>:<k8s-apiserver-port>",
```

Comment:
Should this parameter be automatically generated by Helm rather than exposed to user?

Reply:
Sounds good, I will fix it soon.

Reply:
I find that there is default api server uri(`k8s://kubernetes.default.svc`) which all pods can access. It is really a good choice for spark optimizer to determine spark master parameter. I have tested in Huaweicloud and Googlecloud.

![image](https://github.com/NetEase/amoro/assets/5411862/86c5657d-aead-432b-a5c3-75c4ffaa0719)

![image](https://github.com/NetEase/amoro/assets/5411862/002656b5-1b48-4ee5-b806-7cf89ddfac88)